src/jvmMain/kotlin/io/github/fourlastor/wilds_launcher/operating_system/OperatingSystemGetter.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.fourlastor.wilds_launcher.operating_system
+
+import java.util.*
+
+fun getOperatingSystem() : OperatingSystem {
+    val name = System.getProperty("os.name").lowercase(Locale.getDefault())
```

Comment:
this should be `lowercase()`, some locales do weird things when used for lowercasing, and the default value uses `Locale.ROOT`

Reply:
Oh gotcha, it was actually IntelliJ that recommended this!